### PR TITLE
Update smart proxy plugin dependencies spec files to handle SCL

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/plugins/rubygem-openscap/rubygem-openscap.spec [hunk 1]
```diff
@@ -1,103 +1,97 @@
...
+Release: 2%{?dist}
 Summary: A FFI wrapper around the OpenSCAP library
 Group: Development/Languages
 License: GPLv2+
 URL: https://github.com/OpenSCAP/ruby-openscap
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires: ruby(rubygems)
-Requires: rubygem(ffi) >= 1.0.9
-# require libopenscap.so.8 in an arch neutral way
```
Comment (evgeni) [inline packages/plugins/rubygem-openscap/rubygem-openscap.spec]: wonder if this comment should be kept for clarity?

File: packages/plugins/rubygem-openscap/rubygem-openscap.spec [hunk 2]
```diff
@@ -1,103 +1,97 @@
...
 %dir %{gem_instdir}
+%license %{gem_instdir}/COPYING
 %{gem_libdir}
 %exclude %{gem_cache}
 %{gem_spec}
 
-%files devel
-%{gem_docdir}
-%{gem_instdir}/test/
+%files doc
```
Comment (evgeni) [inline packages/plugins/rubygem-openscap/rubygem-openscap.spec]: should this be added to comps? (devel was never there)

File: packages/plugins/rubygem-route53/rubygem-route53.spec [hunk 3]
```diff
@@ -1,87 +1,107 @@
...
         %{buildroot}%{_bindir}/
 find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
 
 %files
 %dir %{gem_instdir}
 %{_bindir}/route53
-%exclude %{gem_instdir}/.*
-%doc %{gem_instdir}/LICENSE
+%exclude %{gem_instdir}/.gitignore
+%{gem_instdir}/.ruby-gemset
```
Comment (evgeni) [inline packages/plugins/rubygem-route53/rubygem-route53.spec]: exclude

File: packages/plugins/rubygem-route53/rubygem-route53.spec [hunk 4]
```diff
@@ -1,87 +1,107 @@
...
+%{gem_instdir}/.ruby-version
```
Comment (evgeni) [inline packages/plugins/rubygem-route53/rubygem-route53.spec]: exclude

File: packages/plugins/rubygem-ruby-hmac/rubygem-ruby-hmac.spec [hunk 5]
```diff
@@ -66,6 +72,7 @@ cp -a .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
+%{gem_instdir}/Manifest.txt
```
Comment (evgeni) [inline packages/plugins/rubygem-ruby-hmac/rubygem-ruby-hmac.spec]: what is in this file and why is it moved from doc here?

**Comment by ehelms** [comment]
Updated for comments

**Pull request merged**